include/measurement_kit/dns/dns.hpp

```diff
@@ -36,132 +32,10 @@ MK_DEFINE_ERR(MK_ERR_DNS(16), UnsupportedTypeError, "")
 
 Error dns_error(int code); ///< Map evdns code to proper error
 
-/// Available query classes id
```

Comment:
This chunk of code was ugly and also was getting in the way complicating the porting. I prefer strings and also @hellais expressed the same preference. Now I have an excuse to nuke it.